net-misc/purritobin/purritobin-0.1.2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="minimalistic commanline pastebin"
+HOMEPAGE="https://bsd.ac"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/PurritoBin/PurritoBin.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/PurritoBin/PurritoBin/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+	S="${WORKDIR}/PurritoBin-${PV}"
+fi
+
+LICENSE="GPL-2"
```

Comment:
It looks like PurritoBin was released under the terms of the ISC license, not GPLv2.

Reply:
oh, damn, not sure how i missed this, given that i'm the one who wrote purritobin :smiley_cat: